uiowa/uiowa
# Feature: banner backgrounds and collections hotfix

Resolves https://github.com/uiowa/uiowa/issues/3660. 



# How to test
- Try adding a new banner and verify that the Background select appears:

<img width="914" alt="Screen Shot 2021-06-16 at 9 39 25 AM" src="https://user-images.githubusercontent.com/1036433/122239676-befb1e80-ce86-11eb-9c5b-8d2f76c02dd1.png">

- Try adding a collection and verify that the view mode sits above the collection items:

<img width="938" alt="Screen Shot 2021-06-16 at 9 40 07 AM" src="https://user-images.githubusercontent.com/1036433/122239780-d76b3900-ce86-11eb-9c7b-1be3f280519d.png">

- Create a people block and use the person type and person type status filters to person list block and verify that they aren't broken.
